### DISCUSSION TITLE
Lists of domains.

### DISCUSSION THREAD

**Discussion by BobSquarePants** [evilsocket/opensnitch#599] [Q&A] [answered]
Hi,

For a rule we can add a folder containing files with list of domains to block see : 
https://github.com/evilsocket/opensnitch/wiki/block-lists#notes

So there are different type of list

- hosts format
- regular expressions
- IPs
- NETs

Is in a folder can we mix the types ?

If yes I will adapt the wiki :) (happy to contribute to this project ! )

**Comment by gustavo-iniguez-goya** [comment, marked as answer]
> Is in a folder can we mix the types ?

Nope, every type of list must be placed in a different directory.
